consensus/istanbul/backend/backend.go

```diff
@@ -229,13 +229,28 @@ func (sb *backend) checkInSubList(prevHash common.Hash, valSet istanbul.Validato
 }
 
 // getTargetReceivers returns a map of nodes which need to receive a message
```

Comment:
Please describe which nodes will be got the message like below.
```suggestion
// getTargetReceivers returns a map of following nodes which need to receive a message
// - current round committee nodes 
// - next round's committee nodes
```

Reply:
@KimKyungup I'm not sure if it is needed. Because the function is used in making consensus, so only consensus nodes are target receivers.